### ISSUE TITLE
Have collect bail on token expired error

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/cloudmapper#692]
When you get a ClientError TokenExpired, the session has expired so the collect should bail.

**Comment by RoganDawes** [comment]
Yeah, I ran into this as well. My scenario was somewhat different, though, and I'm not sure whether it fits here directly, or should be a different issue entirely.
My IAM identity was given permissions to assume an audit role in the target account, but that role credential/token was only valid for an hour. Collect would take longer than an hour to run, and would consequently hit this condition when the token expired. In those situations, would it not be possible to attempt to refresh the token, and continue, giving up only if the refresh also failed?
